.github/workflows/publish-docker-images.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           flavor: latest=false
           tags: |
             type=ref,event=tag
+            type=semver,pattern=v{{major}}.{{minor}},enable=${{ endsWith(github.ref_name, '.0') }}
```

Comment:
As far as I understand, it will not work if I publish a tag name `v0.27.2` because it does not end with `.0`
Could it be possible to create (maybe with a regexp?) this shorter Docker tag if the git tag finishes by `.` and a number? In the best case, this number could be multiple digits, like `v0.27.2`, but also `v0.27.12`

Reply:
You are right! I had misunderstood the requirements (sorry) -> I thought that it would only be needed if the patch ended with a ".0". 

I have made a new commit that adds an additional workflow step runs the regex matching. It had to be done this way because, (as far as I understand), we can't run pattern matching in an expression. 

The pattern I'm matching to is 'v<number>.<number>.<number>', so that it doesn't catch tags that could look like 'v0.26.1-cloud-team.1', which still end with '.<number>'.

Reply:
Thanks a lot for your reactivity! And the explanations!